client/gutenboarding/onboarding-block/edit.tsx

```diff
@@ -2,38 +2,67 @@
  * External dependencies
  */
 import { __ as NO__ } from '@wordpress/i18n';
-import { TextControl, SelectControl } from '@wordpress/components';
+import { SelectControl } from '@wordpress/components';
 import React from 'react';
 
 /**
  * Internal dependencies
  */
 import { SiteType } from '../store/types';
 import { useOnboardingDispatch, useOnboardingState } from '../store';
+import './style.scss';
 
 export default function OnboardingEdit() {
-	const { siteTitle, siteType } = useOnboardingState();
-	const { setSiteType, setSiteTitle } = useOnboardingDispatch();
+	const { siteTitle, siteTopic, siteType } = useOnboardingState();
+	const { setSiteType, setSiteTopic, setSiteTitle } = useOnboardingDispatch();
+
+	/* @TODO: sets state to control `is-focused` classes for styling */
+	const onBlur = () => {};
+	const onFocus = () => {};
 
 	return (
-		<>
-			<p>{ NO__( "Let's set up your website -- it takes only a moment" ) }</p>
-			{ NO__( 'I want to create a website ' ) }
-			<SelectControl< SiteType >
-				onChange={ setSiteType }
-				options={ [
-					{ label: NO__( 'with a blog.' ), value: SiteType.BLOG },
-					{ label: NO__( 'for a store.' ), value: SiteType.STORE },
-					{ label: NO__( 'to write a story.' ), value: SiteType.STORY },
-				] }
-				value={ siteType }
-			/>
-			{ ( siteType || siteTitle ) && (
-				<>
-					<p>{ NO__( "It's called" ) }</p>
-					<TextControl onChange={ setSiteTitle } value={ siteTitle } />
-				</>
+		<div className="onboarding__questions">
+			<h2>{ NO__( "Let's set up your website – it takes only a moment." ) }</h2>
+
+			<label className="onboarding__question">
+				<span>{ NO__( 'I want to create a website ' ) }</span>
+				<SelectControl< SiteType >
+					onBlur={ onBlur }
+					onChange={ setSiteType }
+					onBlur={ onFocus }
+					options={ [
+						{ label: NO__( 'with a blog.' ), value: SiteType.BLOG },
+						{ label: NO__( 'for a store.' ), value: SiteType.STORE },
+						{ label: NO__( 'to write a story.' ), value: SiteType.STORY },
+					] }
+					value={ siteType }
+					className="onboarding__question-input"
+				/>
+			</label>
+			{ ( siteType || siteTopic ) && (
+				<label className="onboarding__question">
+					<span>{ NO__( 'My site is about' ) }</span>
+					<input
```

Comment:
Using bare `input` instead of `TextControl` since we don't actually need form control looking input.